diff --git a/bishengir/include/bishengir/Dialect/HIVM/IR/HIVMAttrs.td b/bishengir/include/bishengir/Dialect/HIVM/IR/HIVMAttrs.td
index 39fe60d..a000c69 100644
--- a/bishengir/include/bishengir/Dialect/HIVM/IR/HIVMAttrs.td
+++ b/bishengir/include/bishengir/Dialect/HIVM/IR/HIVMAttrs.td
@@ -1069,4 +1069,13 @@ def HIVM_TileMixCubeNumAttr
   }];
 }
 
+def HIVM_TightlyCoupledBufferAttr
+    : HIVM_Attr<"HIVMTightlyCoupledBuffer", "tightly_coupled_buffer"> {
+  let parameters = (ins OptionalParameter<"std::optional<int32_t>">:$id);
+  let assemblyFormat =  "`<` params `>`";
+  let description = [{
+    HIVM tightly coupled buffer for cv.
+  }];
+}
+
 #endif // BISHENGIR_DIALECT_HIVM_IR_HIVMATTRS_TD
